docs/DeveloperGuide.md

```diff
@@ -42,57 +42,64 @@ _Written by: Author name_
 
 ## Acknowledgements
 
-{List here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{List here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
 
 _Written by: Author name_
 
 ## Setting Up the Project
 
 {Detail how to set up the project on one's computer, assuming the software is Intellij IDEA}
 Before setting up the project on your computer, kindly check that you have installed:
+
 * Java JDK 11
 * Intellij IDEA - highly recommended
 
 Firstly, you should fork this repo, before cloning the fork to your computer.
 
 Next,
```

Comment:
There are unexpected/accidental line breaks in these parts of the developer guide, and it overwrites others' documentation. Kindly review and remove these changes, thanks!